tests/gpflow/kernels/test_coregion.py

```diff
@@ -23,23 +23,27 @@
 
 
 class Datum:
-    N1, N2 = 6, 12
+    N1, N2 = 12, 16
     X = [rng.rand(N1, 2) * 1, rng.rand(N2, 2) * 1]
     Y = [
         np.sin(x[:, :1]) + 0.9 * np.cos(x[:, 1:2] * 1.6) + rng.randn(x.shape[0], 1) * 0.8 for x in X
     ]
     label = [np.zeros((N1, 1)), np.ones((N2, 1))]
-    perm = list(range(30))
-    rng.shuffle(perm)
-    Xtest = rng.rand(10, 2) * 10
-    X_augumented = np.hstack([np.concatenate(X), np.concatenate(label)])
-    Y_augumented = np.hstack([np.concatenate(Y), np.concatenate(label)])
+    X_augmented0 = np.hstack([X[0], label[0]])
+    X_augmented1 = np.hstack([X[1], label[1]])
+    X_augmented = np.vstack([X_augmented0, X_augmented1])
+
+    Y_augmented0 = np.hstack([Y[0], label[0]])
+    Y_augmented1 = np.hstack([Y[1], label[1]])
+    Y_augmented = np.vstack([Y_augmented0, Y_augmented1])
+
     # For predict tests
-    X_augumented0 = np.hstack([Xtest, np.zeros((Xtest.shape[0], 1))])
-    X_augumented1 = np.hstack([Xtest, np.ones((Xtest.shape[0], 1))])
-    Ytest = [np.sin(x) + 0.9 * np.cos(x * 1.6) for x in Xtest]
-    Y_augumented0 = np.hstack([Ytest, np.zeros((Xtest.shape[0], 1))])
-    Y_augumented1 = np.hstack([Ytest, np.ones((Xtest.shape[0], 1))])
+    Xtest = rng.rand(10, 2) * 10
+    Xtest_augmented0 = np.hstack([Xtest, np.zeros((10, 1))])
+    Xtest_augmented1 = np.hstack([Xtest, np.ones((10, 1))])
+    Ytest = np.sin(Xtest[:, :1]) + 0.9 * np.cos(Xtest[:, 1:2] * 1.6)
+    Ytest_augmented0 = np.hstack([Ytest, np.zeros((10, 1))])
+    Ytest_augmented1 = np.hstack([Ytest, np.ones((10, 1))])
```

Comment:
test is less brittle when not hard-coding 10 everywhere... why not define N=10 and use that everywhere?